src/main/webapp/modal-overlay.css

```diff
@@ -41,6 +42,27 @@
 
 .modal-body {
   padding: 10px 15px;
+  max-height: 80%;
+}
+
+.modal-body ul {
+  position: static;
+  top: 20px;
+  bottom: 20px;
+  left: 20px;
+  right: 20px;
```

Comment:
just because I made this mistake, we should be careful to test on multiple screens when using pixel values rather than percentages. I'm sure it's fine here though.

Reply:
I spent a lot of time playing with these, but due to some of stacking implementation __(subject to change)__ I had to use ```px``` as opposed to percentages of the anchor.